main/init/sbt/src/mill/main/sbt/SbtBuildGenMain.scala

```diff
@@ -108,13 +108,12 @@ object SbtBuildGenMain
         ```
          */
         val sbtFile = writeTempSbtFileInSbtProjectDirectory(workspace)
-        val commandResult = os.call(
+        os.call(
           (sbtExecutable, "millInitExportBuild"),
           cwd = workspace,
           stdout = os.Inherit
         )
         os.remove(sbtFile)
-        commandResult
```

Comment:
This looks wrong. We now no longer return the result of `os.call` but `os.remove`.

Reply:
IntelliJ IDEA warned me that this variable is not used. I think the return value is not needed here. The variable was added when I first used the `scala.sys.process` APIs and had to judge whether the command was successful from the exit code, but later I refactored this part to use OS-Lib and this is no longer needed.

Reply:
If the return value isn't supposed to transport semantics, let's explicitly make it `: Unit`.

Reply:
I see.